ui/hooks/useAccountTotalFiatBalanceWrapper.ts

```diff
@@ -0,0 +1,16 @@
+import { InternalAccount, isEvmAccountType } from '@metamask/keyring-api';
+import { useAccountTotalFiatBalance } from './useAccountTotalFiatBalance';
+import { useMultichainAccountTotalFiatBalance } from './useMultichainAccountTotalFiatBalance';
+
+export const useAccountTotalFiatBalanceWrapper = (
```

Comment:
I feel like we should not have "Wrapper" in the name here, what about merging this one with `useMultichainAccountTotalFiatBalance` and use the same logic than for multichain selectors?

So we would use `useMultichainAccountTotalFiatBalance` directly, and if it's an EVM account, we fallback to `useAccountTotalFiatBalance`, otherwise we use the new logic from `useMultichainAccountTotalFiatBalance` for non-EVM accounts.

I'm thinking of something like this:
```ts
export const useNonEvmAccountTotalFiatBalance = (
  account: InternalAccount,
): {
  formattedFiat: string;
  totalFiatBalance: string;
  tokensWithBalances: {
    address: string;
    symbol: string;
    decimals: string;
    isERC721: boolean;
    image: string;
  }[];
  loading: boolean;
  orderedTokenList: { iconUrl: string; symbol: string; fiatBalance: string }[];
  error: string | null;
} => {
  ...
  
}

export const useAccountTotalFiatBalance = (
  account: InternalAccount,
  shouldHideZeroBalanceTokens: boolean,
) => {
  if (isEvmAccountType(account.type)) {
    return useAccountTotalFiatBalance(
      account.address,
      shouldHideZeroBalanceTokens,
    );
  }
  return useNonEvmAccountTotalFiatBalance(account);
};
```

Also, the return type of `useNonEvmAccountTotalFiatBalance` should probably be a type too, WDYT?